### DISCUSSION TITLE
blog/nextjs-auth-hoc

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#360] [General]
# blog/nextjs-auth-hoc

Solve problems such as colocation, and error-prone code

https://theodorusclarence-git-hoc-theodorusclarence.vercel.app/blog/nextjs-auth-hoc

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Reply by KhieuDV123** [reply]
Test

**Comment by bsor-dev** [comment]
I'm getting Error: Hydration failed because the initial UI does not match what was rendered on the server. When using with getServerSide together

**Reply by theodorusclarence** [reply]
there must be a content that differs from the prerendered HTML. Do you have any element that is updated in the client side?

**Comment by KhieuDV123** [comment]
Good job
